src/part-guide/concurrency.md

```diff
@@ -22,7 +22,7 @@ do_another_thing();
 
 Each statement is completed before the next one starts[^obs1]. Nothing happens in between those statements[^obs2]. This might sound trivial but it is a really useful property for reasoning about our code. However, it also means we waste a lot of time. In the above example, while we're waiting for `println!("hello!")` to happen, we could have executed `do_another_thing()`. Perhaps we could even have executed all three statements at the same time.
 
-Even where IO[^io-def] happens (printing using `println!` is IO - it is outputting text to the console via a call to the OS), the program will wait for the IO to complete[^io-complete] before executing the next statement. Waiting for IO to complete before continuing with execution *blocks* the program from making other progress. Blocking IO is the easiest kind of IO to use, implement, and reason about, but it is also the least efficient - in a sequential world, the program can do nothing while it waits for the IO to complete.
+Even where IO[^io-def] happens (printing using `println!` is IO - it is outputting text to the console via a call to the OS), the program will wait for the IO to complete[^io-complete] before executing the next statement. Waiting for IO to complete before continuing with execution *blocks* the program from progressing with anything else. Blocking IO is the easiest kind of IO to use, implement, and reason about, but it is also the least efficient. In a sequential world, the program can do nothing while it waits for the IO to complete.
```

Comment:
I prefer the 'making progress' formulation to 'progressing', I feel like 'making progress' in execution is a known idiom (maybe close to a term of art) in the concurrency literature. For the punctuation change, the second clause is an implication of the first so I think it should be either a colon or dash rather than a full stop.